api/swig/python/setup.py.in

```diff
@@ -0,0 +1,35 @@
+import setuptools.command.install
+import shutil
+from distutils.sysconfig import get_python_lib
+
+
+class CompiledLibInstall(setuptools.command.install.install):
```

Comment:
Is this the best/easiest approach? What about a more standard setup.py following https://docs.python.org/3/distutils/setupscript.html ?

Reply:
Will address in version 2.0.2, see: https://github.com/OpenNavigationSurface/BAG/issues/69.